### PR TITLE
Add SearchLy API

### PR DESCRIPTION
**Pull request by AlbertSuarez**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by AlbertSuarez** [comment]
Any update related to this PR?

File: README.md [hunk 1]
```diff
@@ -592,6 +592,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Musikki](https://music-api.musikki.com/reference) | Music | `apiKey` | Yes | Unknown |
 | [Musixmatch](https://developer.musixmatch.com/) | Music | `apiKey` | Yes | Unknown |
 | [Openwhyd](https://openwhyd.github.io/openwhyd/API) | Download curated playlists of streaming tracks (YouTube, SoundCloud, etc...) | `No` | Yes | No |
+| [SearchLy](https://www.github.com/AlbertSuarez/searchly) | Similarities search based on song lyrics | No | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md]: Hi. Can you please link to the docs URL?
```suggestion
| [SearchLy](https://searchly.asuarez.dev/docs/v1) | Similarities search based on song lyrics | No | Yes | Unknown |
```

**Comment by AlbertSuarez** [comment]
Agreed with your suggestion @yannbertrand. All yours now.

**Comment by yannbertrand** [comment]
Thank you!

**Pull request merged**